### PR TITLE
fix: modify knode to leafnode for kosmos-scheduler(v1.26.3) plugins

### PR DESCRIPTION
**Pull request by qiuming520**
#### What type of PR is this?

/kind bug
#### What does this PR do?

modify knode to leafnode for kosmos-scheduler(v1.26.3) plugins

#### Which issue(s) does this PR fix?

Fixes #https://github.com/kosmos-io/kosmos/issues/335
#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: deploy/scheduler/deployment.yaml [hunk 1]
```diff
@@ -74,28 +74,28 @@ data:
...
             disabled:
               - name: "VolumeBinding"
             enabled:
-              - name: "KnodeVolumeBinding"
+              - name: "LeafNodeVolumeBinding"
         pluginConfig:
-          - name: KnodeVolumeBinding
+          - name: LeafNodeVolumeBinding
             args:
               bindTimeoutSeconds: 5
```
Comment (duanmengkk) [inline deploy/scheduler/deployment.yaml]: add a new line

Comment (qiuming520) [reply deploy/scheduler/deployment.yaml]: > add a new line

have done

**Pull request merged**